### DISCUSSION TITLE
LTS on 5.5 - security-release-laravel-61834-7232

### DISCUSSION THREAD

**Discussion by sazo** [laravel/framework#33804] [General]
- Laravel Version: 5.5.44
- PHP Version: 7.2
- Database Driver & Version: MySQL

### Description:
The blog post https://blog.laravel.com/security-release-laravel-61834-7232 is only for security fix for 6.x and 7.x but does not say anything about 5.5? did i mis something here? :)  

Version | Release | Bug Fixes Until | Security Fixes Until
-- | -- | -- | --
5.5 (LTS) | August 30th, 2017 | August 30th, 2019 | August 30th, 2020

**Comment by driesvints** [comment]
Moving this to discussions. I'll get back on this later.

**Comment by driesvints** [comment]
This was already merged on 5.5, we'll tag a release soon.
